lib/internal/bootstrap_node.js

```diff
@@ -307,6 +308,24 @@
     return wrappedConsole;
   }
 
+  function setupInspectorCommandLineAPI() {
+    const inspector = process.binding('inspector');
+    const addCommandLineAPIMethod = inspector.addCommandLineAPIMethod;
+    if (!addCommandLineAPIMethod) return;
+
+    const Module = NativeModule.require('module');
+    const path = NativeModule.require('path');
+    const cwd = tryGetCwd(path);
+
+    const consoleAPIModule = new Module('[consoleAPI]');
+    consoleAPIModule.filename = path.join(cwd, consoleAPIModule.id);
+    consoleAPIModule.paths = Module._nodeModulePaths(cwd);
+
+    addCommandLineAPIMethod('require', function require(request) {
+      return consoleAPIModule.require(request);
+    });
```

Comment:
I agree with @TimothyGu that it should at least use `makeRequireFunction()` so that `require.resolve` and friends exist.